modules/prototype-kit/codebase/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:15.12
+
+# VOLUME /usr/src/app .
+WORKDIR /usr/src/app
+
+COPY . .
```

Comment:
Would this mean I would have to rebuild the image to see an update? I think you can use a volume instead and the sub process (gulp server) should auto reload to show the updated changes.

Reply:
Yes. I'll have another go at mounting the directory as a volume tomorrow, but was getting errors doing so earlier and the image was refusing to build.